Xamarin.Forms.Platform.iOS/Renderers/ButtonRenderer.cs

```diff
@@ -88,6 +92,17 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateImage();
 		}
 
+		void SetControlPropertiesFromProxy()
+		{
+			foreach (UIControlState uiControlState in s_controlStates)
+			{
+				Control.SetTitleColor(UIButton.Appearance.TitleColor(uiControlState), uiControlState); // if new values are null, old values are preserved.
+				Control.SetTitleShadowColor(UIButton.Appearance.TitleShadowColor(uiControlState), uiControlState);
+				Control.SetBackgroundImage(UIButton.Appearance.BackgroundImageForState(uiControlState), uiControlState);
+				Control.SetImage(UIButton.Appearance.ImageForState(uiControlState), uiControlState); // UpdateImage() renders this useless.
```

Comment:
@hartez I chose the custom renderer approach. The code is going to test title color, title shadow color, and background image. It looks like setting image doesn't work because `UpdateImage()` overwrites anything set before. Let me know if the 4th line should be removed.

Note that `UpdateImage()` sets image to null only for the normal state.